### PR TITLE
Debian 11 installation

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

Debian 11 installation

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [x] 22.04.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 22.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-poller/using-packages.md [hunk 1]
```diff
@@ -170,6 +212,13 @@ dnf install -y centreon-poller
 yum install -y centreon-poller
 ```
 
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+```shell
+apt install -y centreon-poller
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-poller/using-packages.md:219]:
```suggestion
apt install -y centreon-poller centreon-poller-centreon-engine
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-central-server/using-packages.md [hunk 2]
```diff
@@ -262,6 +318,13 @@ dnf install -y centreon-central
 yum install -y centreon-central
 ```
 
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+```shell
+apt install -y centreon-central
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-central-server/using-packages.md:325]:
```suggestion
apt install -y centreon-central centreon-poller-centreon-engine
```

File: versioned_docs/version-22.04/installation/installation-of-a-central-server/using-packages.md [hunk 3]
```diff
@@ -266,6 +323,13 @@ dnf install -y centreon-central
 yum install -y centreon-central
 ```
 
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+```shell
+apt install -y centreon-central
```
Comment (Sims24) [inline versioned_docs/version-22.04/installation/installation-of-a-central-server/using-packages.md:330]:
```suggestion
apt install -y centreon-central centreon-poller-centreon-engine
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-central-server/using-packages.md [hunk 4]
```diff
@@ -262,6 +318,13 @@ dnf install -y centreon-central
 yum install -y centreon-central
 ```
 
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+```shell
+apt install -y centreon-central centreon-poller-centreon-engine
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-central-server/using-packages.md:325]:
```suggestion
apt install -y centreon-central
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-poller/using-packages.md [hunk 5]
```diff
@@ -170,6 +212,13 @@ dnf install -y centreon-poller
 yum install -y centreon-poller
 ```
 
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+```shell
+apt install -y centreon-poller centreon-poller-centreon-engine
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/installation-of-a-poller/using-packages.md:219]:
```suggestion
apt install -y centreon-poller
```

File: versioned_docs/version-22.04/installation/installation-of-a-poller/using-packages.md [hunk 6]
```diff
@@ -171,6 +215,13 @@ dnf install -y centreon-poller
 yum install -y centreon-poller
 ```
 
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+```shell
+apt install -y centreon-poller centreon-poller-centreon-engine
```
Comment (Sims24) [inline versioned_docs/version-22.04/installation/installation-of-a-poller/using-packages.md:222]:
```suggestion
apt install -y centreon-poller
```

**Review by Sims24 — DISMISSED** [review]
Ca s'en va et ça revient! C'est fait de tout petits rien ...! 

Merci

**Review by thiuyendang — APPROVED** [review]

File: versioned_docs/version-22.04/installation/web-and-post-installation.md [hunk 7]
```diff
@@ -50,7 +50,7 @@ Then click on **Next**.
 
 Provide the information needed to connect to the database instance.
 
-- **Database Host Address**: If you are using a local database, leave the field blank (the default value is **localhost**). Otherwise, enter the IP address of your remote database.
+- **Database Host Address**: If you are using a local database, leave the field blank (the default value is **localhost**), or, if you are using Debian, enter 127.0.0.1. Otherwise, enter the IP address of your remote database.
```
Comment (lpinsivy) [inline versioned_docs/version-22.04/installation/web-and-post-installation.md:53]: We fixed this in Broker, not necessary to put 127.0.0.1, localhost works correctly

File: versioned_docs/version-22.04/reporting/installation.md [hunk 8]
```diff
@@ -188,6 +188,25 @@ dnf install centreon-bi-server
 yum install centreon-bi-server
 ```
 
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+Add the following external repository (for Java 8):
+
+```shell
+sudo su
```
Comment (lpinsivy) [inline versioned_docs/version-22.04/reporting/installation.md:197]: for other installation you don't put "sudo su"

File: versioned_docs/version-22.04/service-mapping/install.md [hunk 9]
```diff
@@ -20,10 +23,31 @@ Add the Centreon BAM repository, you can find it on the
...
+
 ``` shell
 yum install centreon-bam-server
 ```
 
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+```shell
+sudo apt update && sudo apt install centreon-bam-server
```
Comment (lpinsivy) [inline versioned_docs/version-22.04/service-mapping/install.md:44]: please remove "sudo" command

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/graph-views/install.md [hunk 10]
```diff
@@ -238,6 +238,21 @@ dnf install -y https://yum.centreon.com/standard/22.04/el8/stable/noarch/RPMS/ce
 yum install -y https://yum.centreon.com/standard/22.04/el7/stable/noarch/RPMS/centreon-release-22.04-3.el7.centos.noarch.rpm
 ```
 
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+Pour installer le dépôt Centreon, exécutez la commande suivante:
+
+```shell
+echo "deb https://apt.centreon.com/repository/22.04/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/centreon.list
```
Comment (lpinsivy) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/graph-views/install.md:247]: you missed:
Installez les dépendances suivantes :

```shell
apt update && apt install lsb-release ca-certificates apt-transport-https software-properties-common wget gnupg2
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/web-and-post-installation.md [hunk 11]
```diff
@@ -52,7 +52,7 @@ Puis cliquez sur **Next**.
 
 Fournissez les informations de connexion à l'instance de base de données.
 
-- **Database Host Address** : si vous utilisez une base de données locale, laissez ce champ vide (la valeur par défaut étant **localhost**). Sinon, renseignez l'adresse IP de votre base de données déportée.
+- **Database Host Address** : si vous utilisez une base de données locale, laissez ce champ vide (la valeur par défaut étant **localhost**), ou, si vous utilisez Debian, entrez 127.0.0.1. Sinon, renseignez l'adresse IP de votre base de données déportée.
```
Comment (lpinsivy) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/installation/web-and-post-installation.md:55]: You can remove 127.0.0.1, it's work with localhost

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/reporting/installation.md [hunk 12]
```diff
@@ -201,6 +201,25 @@ dnf install centreon-bi-server
...
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+Ajoutez le dépôt externe suivant (pour Java 8):
+
+```shell
+sudo su
+wget -qO - https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | apt-key add -
+exit
+sudo add-apt-repository --yes https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/
```
Comment (lpinsivy) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/reporting/installation.md:213]: please remove sudo

**Review by lpinsivy — CHANGES_REQUESTED** [review]

**Review by thiuyendang — APPROVED** [review]

**Review by lpinsivy — APPROVED** [review]

**Review by tuntoja — APPROVED** [review]

**Pull request merged**